### PR TITLE
[staging-17.0]:  Merge master to staging-17.0 up to 9bf833fba

### PR DESCRIPTION
**Pull request by darcywong00**
8 of x Merging master to staging-17.0 up to 9bf833fba

I had to resolve some font association in 72aa76f6b55abfa8fa8aab9e6253fdf22995e491 that seemed incomplete on #2512 (.kps didn't assign `OSKFont` and `DisplayFont`)

### CODE REVIEW

File: release/e/eo_plus/eo_plus.keyboard_info
Comment (mcdurdin) [inline release/e/eo_plus/eo_plus.keyboard_info:1]: This metadata needs to move into the .kps.

File: release/sil/sil_tagdal/source/sil_tagdal.kps [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+      <CopyLocation>0</CopyLocation>
+      <FileType>.png</FileType>
+    </File>
+    <File>
+      <Name>welcome\desktop_shift.png</Name>
+      <Description>File desktop_shift.png</Description>
+      <CopyLocation>0</CopyLocation>
+      <FileType>.png</FileType>    <LicenseFile>..\LICENSE.md</LicenseFile>
+
+    </File>
```
Comment (mcdurdin) [inline release/sil/sil_tagdal/source/sil_tagdal.kps:75]:
```suggestion
      <FileType>.png</FileType>
    </File>
```

File: release/sil/sil_tagdal/source/sil_tagdal.kps [hunk 2]
```diff
@@ -70,8 +70,8 @@
       <Name>welcome\desktop_shift.png</Name>
       <Description>File desktop_shift.png</Description>
       <CopyLocation>0</CopyLocation>
-      <FileType>.png</FileType>    <LicenseFile>..\LICENSE.md</LicenseFile>
-
+      <FileType>.png</FileType>
+      <LicenseFile>..\LICENSE.md</LicenseFile>
```
Comment (mcdurdin) [inline release/sil/sil_tagdal/source/sil_tagdal.kps:74]: This shouldn't be here...

Comment (darcywong00) [reply release/sil/sil_tagdal/source/sil_tagdal.kps]: :man_facepalming:

**Pull request merged**